### PR TITLE
Add all Mercury vehicles to common.js #47

### PR DESCRIPTION
**Pull request by hasalassh**
Add all Mercury vehicles to common.js #47

### CODE REVIEW

File: common.js [hunk 1]
```diff
@@ -3198,6 +3198,221 @@ const allSoloCars = {
       'all': ['dp'],
     },
   },
+  'Mercury': {
+    'Maverick & Comet (6-cyl & 8cyl)': {
```
Comment (Bjorn248) [inline common.js:3202]: This seems like a syntax error. A model shouldn't be inside a model.

File: common.js [hunk 2]
```diff
@@ -3198,6 +3198,221 @@ const allSoloCars = {
...
+      'Thunderbird (V6 & TurboCoupe)': {
```
Comment (Bjorn248) [inline common.js:3203]: This is not a Mercury.

File: common.js (hunk 1 above)
Comment (Bjorn248) [inline common.js:3202]: The Maverick is not a Mercury model, but the Comet is

**Review by Bjorn248 — CHANGES_REQUESTED** [review]
Found a few models that are not Mercury. Please check all makes to make sure they are actually Mercury, and not Ford or another make.

**Comment by hasalassh** [comment]
@Bjorn248 Thanks for the review comments. I have done the requested modifications...!

**Comment by hasalassh** [comment]
> Found a few models that are not Mercury. Please check all makes to make sure they are actually Mercury, and not Ford or another make.

Removed few models that are not Mercury from the json...!

**Comment by Bjorn248** [comment]
Thanks

**Comment by hasalassh** [comment]
@Bjorn248 Thanks for the support...!

**Pull request merged**